storage/connect/odbconn.cpp

```diff
@@ -1290,9 +1290,8 @@ bool ODBConn::DriverConnect(DWORD Options)
   HWND    hWnd = (HWND)1;
 #endif  // !__WIN__
   PGLOBAL& g = m_G;
-  PDBUSER dup = PlgGetUser(g);
 
-//if (Options & noOdbcDialog || dup->Remote)
+//if (Options & noOdbcDialog || PlgGetUser(g)->Remote)
```

Comment:
No, I think even commented code should be kept consistent with uncommented code, even if it's not used at the moment for anything. Otherwise you potentially end up with code that you potentially want in some case and end up having to completely reimplement because it's too much work to apply all the necessary changes that were piled into the surrounding code to make the commented code work with it.

Reply:
I understand your concerns. I suggest let's both of us say a big thanks to version control then and happily remove this commented line altogether :-)

Reply:
Done.